README.md

```diff
@@ -1,53 +1,61 @@
 # Book-Library
 
-A simple library where users can store, read and receive books
+An API library where users can store, read and receive books
 
 ## How to run locally
 
 - Clone the project locally
-- cd into the main project
+- cd into the project folder
 - run `npm install` to install dependencies
-  - nodemon
-  - dotenv
-  - express
-- run `npm run devStart` to start local instance using nodemon
+- run `npm run dev:start` to start local instance using nodemon
```

Comment:
The script in the package.json file uses `devStart`

I will update that

Thank you for this correction